frontend/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -141,45 +138,28 @@ export class DashboardComponent implements OnInit {
     this.addDashboardLoading = true
     this.dashboardLocationError = ''
     this.fastenApi.addDashboardLocation(this.dashboardLocation).subscribe((result) => {
-      console.log("Added Remote Dashboard", result)
       this.addDashboardLoading = false
 
       this.modalService.dismissAll()
 
       //reload the page
       window.location.reload()
     }, (error) => {
-      console.log("Error Adding Remote Dashboard", error)
       this.addDashboardLoading = false
       this.dashboardLocationError = error
 
       },
     () => {
-      console.log("Completed Adding Remote Dashboard")
       this.addDashboardLoading = false
     })
   }
 
   public showAddDashboardLocationModal(content) {
     this.dashboardLocation = ''
-    this.modalService.open(content, { ariaLabelledBy: 'modal-basic-title' }).result.then(
-      (result) => {
-        console.log(`Closed with: ${result}`)
-      },
-      (reason) => {
-        console.log(`Dismissed ${reason}`)
-      },
-    );
+    this.modalService.open(content, { ariaLabelledBy: 'modal-basic-title' }).result;
   }
   public showDashboardSettingsModal(content) {
-    this.modalService.open(content, { ariaLabelledBy: 'modal-basic-title' }).result.then(
-      (result) => {
-        console.log(`Closed with: ${result}`)
-      },
-      (reason) => {
-        console.log(`Dismissed ${reason}`)
-      },
-    );
```

Comment:
`.then` is required, otherwise promise will not be fulfilled or executed

Reply:
Added `.catch(console.error)`
https://github.com/fastenhealth/fasten-onprem/pull/462/commits/e4570882a0b28c33899596a8a09980c035fb1e41

I didn't find any other examples except the ones you already tagged in the PR 👍🏻